src/Perpetuum/Services/Channels/ChannelMessageHandler.cs

```diff
@@ -10,25 +10,38 @@ namespace Perpetuum.Services.Channels
 {
     public static class ChannelMessageHandler
     {
-        private const string WELCOME_CHANNEL_NAME = "General chat";
+        private const string WELCOME_CHANNEL_NAME = "Recruitment";
+        private const string HELP_CHANNEL_NAME = "regchannel_help";
         private const string SENDER_CHARACTER_NICKNAME = "New Player Announcer";
 
+        public static void SendNewPlayerTutorialMessage(IChannelManager channelManager, string newCharacterName)
+        {
+            var test = channelManager.Channels.All(channel => channel.Name.ToLower().Contains("help"));
+
+            SendMessage(channelManager, HELP_CHANNEL_NAME, "NewPlayerEnteredTheGame", newCharacterName);
```

Comment:
We should put these magic strings in a central place somewhere.

Reply:
I second the Enum - its an existing pattern in the server/DB where there are enums of "names" that serve as unique keys on some table.
This is how the pre-mademail and these chat messages are setup DB side, so it would be best if these were pulled into an Enum, that way the string value is in one place and will be easier to maintain/debug